src/pubsub/handlers/setApiResponseHandler.js

```diff
@@ -102,4 +104,14 @@ export default class SetApiResponseHandler extends BaseHandler {
     const reportIdString = JSON.stringify({ report: 'Received UserInterest apiResponse parameters' });
     return reportIdString;
   }
+
+  // importing external Api resonse function, which can set the pre-requisite values to external modules
+  setExternalResponse(message) {
+    try {
+      const externalFunction = require('../../../plugins/setExternalApiResponse');
```

Comment:
I want to make sure that this `require` won't break OS when that file does not exist.

Reply:
Yes Jacob, to address your concern, I have tested the usecase where the file required does not exist. 
FCA still starts and behaves as normal with a 'warning' mentioning the module was not found
And since we have handled the code in a try-catch block..the playercreation fails as expected and we get the error message we are throwing from FCA. Basically the intention of requiring this function in the response handler is to handle response so that playerid in the response can be stored at fca side during player creation, which will be used for further calls. This code doesnot tamper with the existing functionality of FCA 
<img width="898" alt="image" src="https://github.com/user-attachments/assets/9a023d62-d917-4314-aaca-5e5f63582d3f">